file-formats/susi/type/zif_aff_susi_v1.intf.abap

```diff
@@ -0,0 +1,164 @@
+      "! Manual maintenance
+      manual                  TYPE ty_maintenance_mode VALUE space,
+      "! <p class="shorttext">Automatic Maintenance (All Objects)</p>
+      "! Automatic maintenance (all objects)
+      automatic               TYPE ty_maintenance_mode VALUE 'A',
+      "! <p class="shorttext">Automatic Maintenance (Basis Authorization Objects Only)</p>
+      "! Automatic maintenance (basis authorization objects only)
+      automatic_basis_objects TYPE ty_maintenance_mode VALUE 'B',
+      "! <p class="shorttext">Application Does Not Require Default Values</p>
+      "! Application does not require default values
+      no_default_values       TYPE ty_maintenance_mode VALUE 'I',
+      "! <p class="shorttext">Application Is Deprecated</p>
+      "! Application is deprecated
+      deprecated              TYPE ty_maintenance_mode VALUE 'D',
+      "! <p class="shorttext">Application Is Obsolete</p>
+      "! Application is obsolete
+      obsolete                TYPE ty_maintenance_mode VALUE 'O',
+    END OF co_maintenance_mode,
+
+    "! <p class="shorttext">Maintenance Status</p>
+    "! Maintenance status
+    BEGIN OF co_maintenance_status,
+      "! <p class="shorttext">No Default</p>
+      "! No default
+      no_default             TYPE ty_maintenance_status VALUE 'X',
+      "! <p class="shorttext">Default With Field Values</p>
+      "! Default with field values
+      default_with_values    TYPE ty_maintenance_status VALUE 'Y',
+      "! <p class="shorttext">Default Without Field Values</p>
+      "! Default without field values
+      default_without_values TYPE ty_maintenance_status VALUE 'V',
+      "! <p class="shorttext">Default Inactive</p>
+      "! Default inactive
+      inactive_values        TYPE ty_maintenance_status VALUE 'I',
+    END OF co_maintenance_status.
+
+  TYPES:
+    "! <p class="shorttext">Application Name</p>
+    "! Name of application
+    ty_appl_name   TYPE c LENGTH 132,
+    "! <p class="shorttext">Application Type</p>
+    "! Type of application
+    ty_type        TYPE c LENGTH 80.
+
+  TYPES:
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    "! $required
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Application Name</p>
+      "! Application name of authorization defaults
+      "! $required
+      application_name TYPE ty_appl_name,
+      "! <p class="shorttext">Application Type</p>
+      "! Type of authorization defaults
+      "! $required
+      application_type TYPE ty_type,
+      "! <p class="shorttext">Maintenance Mode</p>
+      "! Maintenance mode
+      "! $values {@link zif_aff_susi_v1.data:co_maintenance_mode}
+      "! $default {@link zif_aff_susi_v1.data:co_maintenance_mode.manual}
+      "! $showAlways
+      maintenance_mode TYPE ty_maintenance_mode,
+      "! <p class="shorttext">Description</p>
+      "! Description of authorization default values
+      description      TYPE string,
```

Comment:
Somehow my comment got lost: Does it make sense to rename `description` (which I understand as long text) to something other (e.g., `documentation`? This might help users can differentiate it to `header.description` which is the object's description (short text)

Reply:
I see, but both functionalities are not really used in the past and I do not think this will change in the future. It is just an opportunities to describe/document/inform about the authorization defaults at all and then special for each object. As long as the fields "Description" are available in two different sections, it should not be a problem to distinguish both(I hope :-) )

Reply:
OK, I see. I just see that this might be misleading for users to have two fields on the same "level" which are called description (even if they are in header and general information. 

However, if you prefer to stick to descriptions and you don't think this might be misleading for users, just let me know. Then I will merge the changes as it is. It can also be changed in the future (if needed) by increasing the file format version.

Reply:
ok, thank you. I would like to ask you to marge the changes.